Sources/SwiftTypeAdoptionReporter/FastStrategy.swift

```diff
@@ -230,54 +169,113 @@ extension FastStrategy: SyntaxVisitor {
         }
     }
 
-    public func visit(_ node: UnknownSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: UnknownSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: InOutExprSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: InOutExprSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: AssignmentExprSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: AssignmentExprSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: TypeExprSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: TypeExprSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: TypeAnnotationSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: TypeAnnotationSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: TypeInitializerClauseSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: TypeInitializerClauseSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: TypealiasDeclSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: TypealiasDeclSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: TypeInheritanceClauseSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: TypeInheritanceClauseSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: ImportDeclSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: ImportDeclSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: FunctionSignatureSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: FunctionSignatureSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: AsTypePatternSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: AsTypePatternSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
-    public func visit(_ node: AsExprSyntax) -> SyntaxVisitorContinueKind {
+    override public func visit(_ node: AsExprSyntax) -> SyntaxVisitorContinueKind {
         return .skipChildren
     }
 
+    // MARK: - Private
```

Comment:
Just moved this section of code from above to keep all `private` properties/functions together